gapic-common/lib/gapic/paged_enumerable.rb

```diff
@@ -83,7 +84,7 @@ def each
 
       each_page do |page|
         page.each do |obj|
-          obj = @format_resource.call obj if @format_resource
+          obj = @format_resource.call obj, page.operation if @format_resource
```

Comment:
```suggestion
          obj = @format_resource.call obj if @format_resource
```

Reply:
I kinda think the `@format_response.call` should be pushed down to `Page#each`.

Reply:
ok, moved.